### ISSUE TITLE
get_chunk_aabb takes a lot of time to complete

### ISSUE THREAD

**Issue by Duroxxigar** [Zylann/godot_heightmap_plugin#155]
**Describe the bug**
The method **get_chunk_aabb** takes anywhere from 40 ms - 80 ms to complete. It is dependent on the chunk size on how long it takes. I'm currently at 40 ms with a chunk size of 64.

The size of the terrain is 3000x3000 (Yes, I know this is large)

**Screenshots**
![image](https://user-images.githubusercontent.com/16636969/79641638-c893d400-8166-11ea-9f98-c5959c2ce9af.png)

**Environment**
- OS: Windows 10
- GTX 1060 6GB
- Godot version 3.2.1
- Plugin version: 1.2
- Renderer used: GLES3

**Comment by Zylann** [comment]
My profiling looks quite different
![image](https://user-images.githubusercontent.com/1311555/79644964-38747f80-81a4-11ea-84a9-09d02297f23d.png)

**Comment by Duroxxigar** [comment]
I have 3 layers and 1 is at 1k, the other 2 are at 500.

**Comment by Zylann** [comment]
Those distances are really high, you should tune them down. Some improvements can be made to increase performance but right now such sizes are not recommended. If you halve view distances it should be 4 times faster.

**Comment by Duroxxigar** [comment]
Yeah - I completely forgot about the detail layers having this property. I'll touch basis with the environment team. Thanks a bunch @Zylann
